docs/pages/careers/developer-advocate.md

```diff
@@ -1,116 +1,111 @@
 # Developer Advocate
```

Comment:
I opted to keep the title open-ended, but we could potentially change this to something like `Content Developer` (for the content path) and/or `DX Engineer` (for the docs infra path)